code-challenges.test.js

```diff
@@ -38,11 +87,49 @@ const fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
 
 // b) Create the function that makes the test pass.
 
+// Pseudocode:
+// 1. Create a function called onlyOddNumbers
+// 2. The parameter will be called array, because it is taking an array input
+// 3. Create a return statement using the .filter hiher order function
+//      3a. Use the value declaration to start creating logic within the filter method
+//      3b. We will check if the value is an odd number using modulo to strictly equal 1
+//      3c. Then we will want to use the && operator to make sure the value is a number using typeof
+//      3d. Lastly, we will do the same, but for negative odd number values using the || (OR) statement
+// 4. Now that we are able to create the array of values of only negative numbers, we will want to sort the array
+//      4a. We will create a new variable called lowestToHighest and plug the array into it and use the .sort method
+// 5. The return function from step 3 will return the array with only odd numbers and in order
+
+const onlyOddNumbers = (array) => {
+    let lowestToHighest = array.sort((a,b) => a - b)
+    
+    return lowestToHighest.filter(value => (value % 2 === 1 && typeof value === "number") || value % 2 === -1 && typeof value === "number")
```

Comment:
Excellent work on this problem!!  Keep in mind, an odd number is any number that is NOT divisible by 2.  This could make for a simple refactor  to account for negative numbers using less code.